journal/src/test/scala/com/evolutiongaming/kafka/journal/JournalSpec.scala

```diff
@@ -675,7 +675,7 @@ object JournalSpec {
             val partitionOffset = PartitionOffset(partition, record.offset)
             EventRecord(action, event.map(eventualWrite(_).get), partitionOffset, events1.metadata)
           }
-          copy(events = events.enqueue(batch.toList), offset = offset.some)
+          copy(events = events.enqueueAll(batch.toList), offset = offset.some)
```

Comment:
@mr-git could you please elaborate this change

Reply:
`enqueue` is deprecated with suggestion to use `enqueueAll`:
https://github.com/scala/scala/blob/a5270194b6eec9678a79e2f88a83d83aa6b92de1/src/library/scala/collection/immutable/Queue.scala#L152